samples/minecraft/sample.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: mps.playfab.com/v1alpha1
+kind: GameServerBuild
+metadata:
+  name: gameserverbuild-sample-minecraft
+spec:
+  titleID: "1E03" # required
+  buildID: "85ffe8da-c82f-4035-86c5-9d2b5f42d6f8" # must be a GUID
+  standingBy: 2 # required
+  max: 4 # required
+  crashesToMarkUnhealthy: 5 # optional, default is 5. It is the number of crashes needed to mark the build unhealthy
+  portsToExpose:
+    - 25565
+  template:
+    spec:
+      containers:
+        - image: rnjohn/minecraft-server-test
```

Comment:
image should be tagged. Is this on Docker Hub? It would be better if the registry is explicit

Reply:
Yes, the image is on [Docker Hub](https://hub.docker.com/repository/docker/rnjohn/minecraft-server-test), I added the latest tag